frontend/src/authentication/components/MaintenanceGuard.js

```diff
@@ -0,0 +1,63 @@
+import PropTypes from 'prop-types';
+import { useEffect, useState } from 'react';
+import { isModuleEnabled, ModuleNames } from 'utils';
+import { useClient, useGroups } from 'services';
+import { LoadingScreen, NoAccessMaintenanceWindow } from 'design';
+import { getMaintenanceStatus } from '../../modules/Maintenance/services';
```

Comment:
You can also write modules/Maintenance/services to import

Reply:
Because the `MaintenanceGuard` wraps the entire DefaultLayout and all the child paths I thought made sense to keep along side the other `...Guard.js` files

And then keep the actual React files that are about the actual functionality of starting / stopping maintenance in the modules child directory

Reply:
I am not super strong on my opinion though on the above - so if you think best to handle differently do let me know

Reply:
@dlpzx let me know thoughts when you get some time - I will merge this tomorrow unless any additional comments to address, thanks